.circleci/config.yml

```diff
@@ -227,12 +230,26 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v[0-9].*/
-#      - build-multiarch-docker:
+#      - macos:
+#          name: macos-diag
 #          filters:
 #            branches:
-#              ignore: /.*/
+#              only: /.*/
 #            tags:
-#              only: /^v[0-9].*/
+#              only: /.*/
+#      - valgrind:
```

Comment:
Do we need this?

Reply:
This is for running valgrind/macos from a PR (like, un-commenting for testing, then re-commenting before merge).